jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/CodeReviewCommentController.java

```diff
@@ -86,11 +86,28 @@ public void initBinder(WebDataBinder binder) {
     @RequestMapping(value = "/reviews/{postId}/json", method = RequestMethod.GET)
```

Comment:
I remove json postfix and add Accept parameter with 'application/json' value to headers of all AJAX requests in CodeReview module. Is that what you mean?